docs/.pages

```diff
@@ -0,0 +1,7 @@
+nav:
+    - Introduction
+    - Architecture
+    - Installation
+    - Troubleshooting
+    - For Developer
+    - ...
```

Comment:
Is it possible to skip the `Home` section in the resulting web page?
![image](https://github.com/bechtleav360/Maverick.UserProfileService/assets/84098418/a6e8f712-27c0-4e79-858e-2ac10dbb6a72)

Reply:
I using now the page **Opening Marks** as first page.